frontend/src/components/search/components/common/filter-dropdown/filter-dropdown.tsx

```diff
@@ -0,0 +1,53 @@
+import { FC } from 'common/types/types';
+import { useState } from 'hooks/hooks';
+import { IconName } from 'common/enums/components';
+import { Icon } from 'components/common/common';
+
+import styles from './styles.module.scss';
+
+type Props = {
+  filter: string;
+  onChangeFilter: (value: string) => void;
+  title: string;
+  options: {
+    text: string;
+    value: string;
+  }[];
+};
+
+const FilterDropdown: FC<Props> = ({ filter, onChangeFilter, title, options }) => {
+  const [toggleDropdown, setToggleDropdown] = useState(false);
+
+  const onHandleToggleDropdown = (): void => setToggleDropdown(!toggleDropdown);
+  const onHandleChangeFilter = (e: React.FormEvent<HTMLInputElement>): void => onChangeFilter(e.currentTarget.value);
+
+  return (
+    <>
+      <div className={styles['filter-dropdown']}>
+        <div className={styles['filter-dropdown-title']} onClick={onHandleToggleDropdown}>
+          {title}
+          <Icon name={toggleDropdown ? IconName.ARROW_UP : IconName.ARROW_DOWN} color="gray" />
```

Comment:
How would you change this `gray` color when we have a toggle dark/light mode feature?

I think we need to have an object with variables for this

Reply:
Or style color of this in scss file